package.json

```diff
@@ -13,7 +13,7 @@
     "lint-es": "eslint --cache --cache-location '.eslintcache/' --ext .js --ext .jsx --max-warnings=0 src e2e",
     "format": "node prettier.js write",
     "format-check": "node prettier.js lint",
-    "start": "razzle start --inspect --inspect-port 9230",
+    "start": "cross-env GOOGLE_APPLICATION_CREDENTIALS=./secrets.json razzle start --inspect --inspect-port 9230",
```

Comment:
Denne env variabelen bør sikkert settes på en annen måte

Reply:
Hvor brukes denne?

Reply:
Denne brukes til å sette client og server secrets for autentisering mot Google analytics management API. Optimalt bør disse verdiene settes som env variabler i de forskjellige miljøene slik at vi slipper secrets.json